client/src/App.tsx

```diff
@@ -12,59 +13,59 @@ import SignIn from './pages/SignIn';
 
 const App: React.FC = () => {
   return (
-    <Router>
-      <div className="App">
-        <nav>
-          <ul>
-            <li>
-              <Link to="/">Home</Link>
-            </li>
-            <li>
-              <Link to="/catalog">Equipment Catalog</Link>
-            </li>
-            <li>
-              <Link to="/equipmentbrowser">Equipment Browser</Link>
-            </li>
-            <li>
-              <Link to="/memberdashboard">MemberDashboard</Link>
-            </li>
-            <li>
-              <Link to="/admin/dashboard">Admin Dashboard</Link>
-            </li>
-            <li>
-              <Link to="/admin/equipment-request">Equipment Request</Link>
-            </li>
-            <li>
-              <Link to="/signin">Sign In</Link>
-            </li>
-          </ul>
-        </nav>
+    <React.StrictMode>
+      <Router>
+        <div className="App">
+          <nav>
+            <ul>
+              <li>
+                <Link to="/">Home</Link>
+              </li>
+              <li>
+                <Link to="/catalog">Equipment Catalog</Link>
+              </li>
+              <li>
+                <Link to="/equipmentbrowser">Equipment Browser</Link>
+              </li>
+              <li>
+                <Link to="/memberdashboard">MemberDashboard</Link>
+              </li>
+              <li>
+                <Link to="/admin/dashboard">Admin Dashboard</Link>
+              </li>
+              <li>
+                <Link to="/admin/equipment-request">Equipment Request</Link>
+              </li>
+              <li>
+                <Link to="/signin">Sign In</Link>
+              </li>
+            </ul>
+          </nav>
 
-        <Switch>
-          <Route path="/admin/dashboard">
-            <AdminDashboard />
-          </Route>
-          <Route path="/catalog">
-            <Catalog />
-          </Route>
-          <Route path="/equipmentbrowser">
-            <EquipmentBrowser />
-          </Route>
-          <Route path="/memberdashboard">
-            <MemberDashboard />
-          </Route>
-          <Route path="/admin/equipment-request">
-            <RequestSummary />
-          </Route>
-          <Route path="/signin">
-            <SignIn />
-          </Route>
-          <Route path="/">
-            <Home />
-          </Route>
-        </Switch>
-      </div>
-    </Router>
+          <Helmet
+            titleTemplate="TUTV | %s"
+            defaultTitle="TUTV Equipment Borrowing"
+          >
+            <meta
```

Comment:
We should make sure to also define this information in the static HTML stored in `public/index.html`, so that document meta information can still be served to agents that don't run JavaScript. The meta description is used almost exclusively by search engines/crawlers, so it makes sense to have it be a part of the initial bundle from the server.

Reply:
Yeah, I chose not to do that because this website won't be visible to search engines. But I'll add something sensible in the .html.